Sources/Host/POSIX/File.cpp

```diff
@@ -54,5 +55,28 @@ ErrorCode File::pread(ByteVector &buf, uint64_t &count, uint64_t offset) {
 
   return _lastError = kSuccess;
 }
+
+ErrorCode File::createDirectory(std::string const &path, uint32_t flags) {
+  size_t pos = 0;
+
+  // Each parent directory must be created individually, if it does not exist
+  while ((pos = path.find('/', pos + 1)) != std::string::npos) {
```

Comment:
Why is the initialization not required? You have to tell find to start at the beginning of the string.